lib/mgmt_msg.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * March 6 2023, Christian Hopps <chopps@labn.net>
+ *
+ * Copyright (c) 2023, LabN Consulting, L.L.C.
+ *
+ */
+#ifndef _MGMT_MSG_H
+#define _MGMT_MSG_H
+
+#include "stream.h"
+#include "thread.h"
+
+#define MGMT_MSG_MARKER (0x4D724B21u) /* ASCII - "MrK!"*/
+
+struct mgmt_msg_state {
+	struct stream *ins;
+	struct stream *outs;
+	struct stream_fifo inq;
+	struct stream_fifo outq;
+	uint64_t nrxm;		/* number of received messages */
+	uint64_t nrxb;		/* number of received bytes */
+	uint64_t ntxm;		/* number of sent messages */
+	uint64_t ntxb;		/* number of sent bytes */
+	size_t max_read_buf;  /* should replace with max time value */
+	size_t max_write_buf; /* should replace with max time value */
+	size_t max_msg_sz;
+	char *idtag; /* identifying tag for messages */
```

Comment:
minor: you have consistent comment indentation everywhere in this file except these 3 lines.

Reply:
I've fix the middle ones, the last one is how clang-format wants it, so I'll leave it. :)